webrender/src/platform/windows/font.rs

```diff
@@ -17,8 +18,11 @@ lazy_static! {
     };
 }
 
+#[allow(dead_code)]
```

Comment:
We don't use the gdi_gamma_lut yet. We will once https://github.com/servo/webrender/pull/770 lands.